src/execution/logging.js

```diff
@@ -0,0 +1,32 @@
+/* @flow */
+/**
+ *  Copyright (c) 2015, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+export const TAG = {
```

Comment:
What's the purpose of logging `RESOLVER_ERROR` or `SUBTREE_ERROR ` if you've already got the `exeContext.errors` Array?